jekyll/_cci2/contexts.md

```diff
@@ -54,11 +54,16 @@ To rerun a job and use the context, it **must** be rerun from the Workflows page
 
 ## Environment Variable Usage 
 
-Environment variables are considered in a specific order, as follows:
-1. Project-level environment variables set on the Project Settings page.
-2. Context environment variables (assuming the user has access to the Context).
-3. Environment variables set with the `environment` key of a `run` step.
-4. Environment variables declared inside a shell command in a `run` step, for example `FOO=bar make install`.
+Environment variables are used according to a specific precedence order, as follows:
+
+1. Environment variables declared inside a shell command in a `run` step, for example `FOO=bar make install`.
+2. Environment variables set with the `environment` key for job.
+3. Environment variables set with the `environment` key for a container.
+4. Context environment variables (assuming the user has access to the Context). See the [Contexts]( {{ site.baseurl }}/2.0/contexts/) documentation for instructions.
+5. Project-level environment variables set on the Project Settings page.
+6. CircleCI environment variables.
```

Comment:
Aren't all of these CircleCI environmental variables? Please disambiguate :)

Reply:
We are missing one item in this list. You can set an `environment` on a `run` step: https://circleci.com/docs/2.0/configuration-reference/#run

I beleive this should be number 2 in the list, but I'm not certain.

Reply:
thanks! I've clarified that last line for Chris and added a new item 2 per Marc, will wait for review from @AlexanderMann or @notnoopci to confirm the ordering of the list